### PR TITLE
Add DDC (082) as subject

### PR DESCRIPTION
**Pull request by TobiasNx**
(#1191)

### CODE REVIEW

File: src/main/resources/alma/maps.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <xi:include href="maps/alma-institution-code-to-isil.xml" />
   <xi:include href="maps/alma-iz-code-to-isil.xml" />
   <xi:include href="maps/rswk-indicator.xml" />
-
+	<filemap name="deweyLabels" files="src/main/resources/alma/maps/deweyLabels.tsv"/>
   <!--
```
Comment (dr0i) [inline src/main/resources/alma/maps.xml:8]: indentation

File: src/main/resources/alma/maps.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <xi:include href="maps/alma-institution-code-to-isil.xml" />
   <xi:include href="maps/alma-iz-code-to-isil.xml" />
   <xi:include href="maps/rswk-indicator.xml" />
-
+	<filemap name="deweyLabels" files="src/main/resources/alma/maps/deweyLabels.tsv"/>
```
Comment (dr0i) [inline src/main/resources/alma/maps.xml:7]: don't reference it like this. Also, we only need the test file. Analog to https://github.com/hbz/lobid-resources/blob/master/src/main/resources/morph-hbz01-to-lobid.xml#L7685 , should work.

File: src/main/resources/alma/maps/deweyLabels.tsv [hunk 3]
```diff
@@ -0,0 +1,45 @@
+004.62	Kopplungs- und Kommunikationsprotokolle
```
Comment (dr0i) [inline src/main/resources/alma/maps/deweyLabels.tsv:1]: don't put it into the repo, for when copy the "other file" to this location this other file would be overriden everytime we pull.

**Review by dr0i — CHANGES_REQUESTED** [review]
Added some comments inline.
Besides this - great to say hello to ddc :)

File: src/main/resources/alma/maps.xml (hunk 2 above)
Comment (TobiasNx) [reply src/main/resources/alma/maps.xml:7]: @dr0i Thisdoes not work, i tried it like this before. The reason is, that in  https://github.com/hbz/lobid-resources/blob/master/src/main/resources/morph-hbz01-to-lobid.xml#L7685 the deweyLabel file is at the same level als the morph. We need to transfer the  map file at least in the maps folder as the other maps..

File: src/main/resources/alma/maps/deweyLabels.tsv (hunk 3 above)
Comment (TobiasNx) [reply src/main/resources/alma/maps/deweyLabels.tsv:1]: As I said at the moment we do not have any deweyLabels.tsv in the maps folder. Either we use this here or you have to create a new file on the server.

Comment (dr0i) [reply src/main/resources/alma/maps/deweyLabels.tsv]: " we do not have any deweyLabels.tsv in the maps folder"
but we have - in the test-folder. The files there are also available when calling them from "src".
See add11eeaf56eac75cae0c9c230c8b40b2750a556.

**Pull request merged**